lua/lspconfig/tflint.lua

```diff
@@ -0,0 +1,22 @@
+local configs = require "lspconfig/configs"
+local util = require "lspconfig/util"
+
+configs.tflint = {
+  default_config = {
+    cmd = {"tflint", "--langserver"},
+    filetypes = {"terraform"},
+    root_dir = util.root_pattern(".terraform", ".git", ".tflint.hcl")
```

Comment:
Why is `.git` preferred over `.tflint.hcl`?

Reply:
I didn't know there is a order of precedence in `util.root_pattern`. If there is I think it should be `".tflint.hcl", ".terraform", ".git"`. What do you think @lithammer ?

Reply:
Sorry I forgot that the search is breadth first (and not depth first). So the order is irrelevant.